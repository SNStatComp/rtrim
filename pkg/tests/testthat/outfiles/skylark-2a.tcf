FILE outfiles/skylark.dat
TITLE  skylark-2a
NTIMES 8
NCOVARS 2
LABELS
Habitat
Cov2
End
MISSING 999
WEIGHT Absent

model 2
covariates 1
weighting off
overdisp on
serialcor on
changepoints 1 2 3 4 5 6 7


