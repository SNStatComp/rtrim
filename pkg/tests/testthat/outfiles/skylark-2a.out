 TRIM 3.61 :  TRend analysis and Indices for Monitoring data 
 STATISTICS NETHERLANDS 

 Date/Time: 25-8-2016 17:58:48

 Title :    skylark-2a

 The following  5 variables have been read from file: 
 Z:\Users\patrick\Work\CBS\TRIM\wine\skylark.dat

 1. Site             number of values:    55
 2. Time             number of values:     8
 3. Count            missing =        999
 4. HABITAT          number of values:  2
 5. COV2             number of values:  4

 Number of sites without positive counts (removed)       0

 Number of observed zero counts           0
 Number of observed positive counts     202
 Total number of observed counts        202
 Number of missing counts               238
 Total number of counts                 440

 Total count                         2536.0

 Sites containing more than 10% of the total count 
   Site Number  Observed Total    %   
         3        431.0         17.0
        37        266.0         10.5
        40        624.0         24.6

 Time Point Averages 
   TimePoint  Observations  Average   Index  
       1              25       8.52    1.00
       2              20       8.10    0.95
       3              30      10.90    1.28
       4              30      11.27    1.32
       5              28      12.75    1.50
       6              29      14.66    1.72
       7              22      17.00    2.00
       8              18      18.89    2.22

 RESULTS FOR MODEL: Linear Trend
 --------------------------------
   Effects of covariate(s)
    HABITAT
   Changes in Slope at Timepoints
      1     2     3     4     5     6     7 

 ESTIMATION METHOD = Generalised Estimating Equations


  Total time used:  4.03 seconds

  Estimated Overdispersion     =  1.162
  Estimated Serial Correlation =  0.227

 GOODNESS OF FIT
  Chi-square                154.50, df    133, p 0.0979
  Likelihood Ratio          159.64, df    133, p 0.0575
  AIC (up to a constant)   -106.36

 WALD-TEST FOR SIGNIFICANCE OF COVARIATES 
 Covariate      Wald-Test  df   p  
  HABITAT          21.55    7  0.0030

 WALD-TEST FOR SIGNIFICANCE OF CHANGES IN SLOPE 
 Changepoint    Wald-Test  df   p  
      1            10.27    2  0.0059
      2             9.18    2  0.0102
      3             3.08    2  0.2143
      4             1.54    2  0.4637
      5             1.64    2  0.4413
      6             0.89    2  0.6419
      7             0.01    2  0.9927

 PARAMETER ESTIMATES 

 Effects of Covariates on Slope for Time Intervals
 from upto
    1    2
               Additive      std.err.   Multiplicative  std.err.
 Constant       -0.2165       0.1991         0.8053      0.1604

 Covariate  1
 ------------
 Category   2   -0.1445       0.2324         0.8655      0.2011

 from upto
    2    3
               Additive      std.err.   Multiplicative  std.err.
 Constant       -0.1616       0.2207         0.8508      0.1878

 Covariate  1
 ------------
 Category   2    0.4216       0.2480         1.5244      0.3781

 from upto
    3    4
               Additive      std.err.   Multiplicative  std.err.
 Constant       -0.1201       0.2195         0.8869      0.1946

 Covariate  1
 ------------
 Category   2    0.1094       0.2336         1.1156      0.2606

 from upto
    4    5
               Additive      std.err.   Multiplicative  std.err.
 Constant       -0.2410       0.2260         0.7859      0.1776

 Covariate  1
 ------------
 Category   2    0.3882       0.2389         1.4744      0.3522

 from upto
    5    6
               Additive      std.err.   Multiplicative  std.err.
 Constant        0.2179       0.2249         1.2434      0.2797

 Covariate  1
 ------------
 Category   2   -0.1298       0.2366         0.8783      0.2078

 from upto
    6    7
               Additive      std.err.   Multiplicative  std.err.
 Constant       -0.1153       0.2180         0.8911      0.1943

 Covariate  1
 ------------
 Category   2    0.2139       0.2301         1.2385      0.2850

 from upto
    7    8
               Additive      std.err.   Multiplicative  std.err.
 Constant       -0.0849       0.2330         0.9186      0.2140

 Covariate  1
 ------------
 Category   2    0.1720       0.2459         1.1876      0.2920


 Time INDICES
  Time     Model     std.err.     Imputed    std.err.
     1     1                       1
     2     0.7281     0.0751       0.7234     0.0751
     3     0.8411     0.0846       0.8422     0.0848
     4     0.8119     0.0835       0.8145     0.0838
     5     0.8757     0.0886       0.8765     0.0888
     6     0.9771     0.0987       0.9792     0.0990
     7     1.0420     0.1068       1.0433     0.1071
     8     1.1106     0.1155       1.1219     0.1164

 TIME TOTALS
  Time      Model    std.err.     Imputed    std.err.
     1        526          44          526          44
     2        383          36          380          36
     3        443          30          443          30
     4        427          28          428          28
     5        461          29          461          29
     6        514          29          515          29
     7        548          34          548          34
     8        585          38          590          38

 OVERALL SLOPE MODEL: Moderate increase (p<0.01) **
  Additive      std.err.   Multiplicative  std.err.
    0.0363       0.0134         1.0370      0.0139

 OVERALL SLOPE IMPUTED+(recommended): Moderate increase (p<0.01) **
  Additive      std.err.   Multiplicative  std.err.
    0.0376       0.0134         1.0384      0.0140


